### DISCUSSION TITLE
Cannot see other options for schema compare

### DISCUSSION THREAD

**Discussion by nivibaliga** [dbeaver/dbeaver#12904] [DBeaver Development]
<img width="577" alt="Screen Shot 2021-06-18 at 3 13 02 PM" src="https://user-images.githubusercontent.com/85903395/122616490-b00f9a00-d047-11eb-9aa1-2778d55d8c3e.png">

Hi. I am trying to use DBeaver to initiate a schema compare and generate the DDL scripts. However, when I click on the "Compare/Migrate" option, I only see the option to perform a "Simple Structure Compare" (as in screenshot) which only generates a tabular report. Please guide.

**Comment by LonwoLonwo** [comment]
Hello @nivibaliga 
Schema compare is the [DBeaver EE](https://dbeaver.com/) feature https://github.com/dbeaver/dbeaver/wiki/Schema-compare
You can download DBeaver EE and activate the [trial license](https://dbeaver.com/trial/) to see how it works.
